docs/VTOL.md

```diff
@@ -113,10 +113,10 @@ set mc_iterm_relax = RPY
 save
 ```
 
-# STEP1: Configuring as a normal fixed-wing in Profile 1
+# STEP 1: Configuring as a normal fixed-wing in Profile 1
```

Comment:
You've changed `STEP1` to `STEP 1`. But the other steps remain without a space. There are also parts in the document that refer to `STEP1` where the space hasn't been added. I don't mind if it has the space or not. But it should be consistent throughout the document.